docs/sources/mimir/release-notes/v2.10.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Grafana Mimir version 2.10 release notes"
+menuTitle: "V2.10 release notes"
+description: "Release notes for Grafana Mimir version 2.10"
+weight: 1000
+---
+
+# Grafana Mimir version 2.10 release notes
+
+Grafana Labs is excited to announce version 2.10 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- The ruler endpoint `/api/v1/rules` now accepts `file`, `ruler_group` and `rule_name` parameters to filter the results.
+- The Cardinality API endpoint `/api/v1/cardinality/label_values` now accepts a `count_method` parameter, which can be set to `active` to only **count series that are considered _active_** according to the `-ingester.active-series-metrics-idle-timeout` flag setting rather than counting all inmemory series.
+- The internal data structure for labels was changed, reducing the memory consumption. Expect ingesters to use around 15% less memory with this change, depending on the pattern of labels used, number of tenants, etc.
+- The memory usage of the Active Series Tracker in the ingester was improved.
+- All components now support buffered logging that can be enabled through the `-log.buffered` CLI flag. This should reduce contention and resource usage under heavy usage patterns.
```

Comment:
This should be mentioned among experimental features, below.

Reply:
The buffered logger isn't marked as experimental right now, should it be?